packages/api/src/clickhouse/index.ts

```diff
@@ -1053,7 +1054,7 @@ const buildEventSeriesQuery = async ({
   endTime,
   field,
   granularity,
-  groupBy,
+  groupBys,
```

Comment:
We should continue to call this `groupBy`, since everywhere else assumes a groupBy is an array

Reply:
sounds good